kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -20,6 +20,4 @@ popd
 # Clean cache, will be populated again when yarn runs next time
 yarn cache clean
 
-<% if File.exist?(@build_base.join(KS_PART_DIR, "post", "npm_registry_cleanup.ks.erb")) %>
-  <%= render_partial "post/npm_registry_cleanup" %>
-<% end %>
+<%= render_partial_if_exist("/build/optional_kickstart_partials/npm_registry_cleanup.ks.erb") %>
```

Comment:
Is it safe to hardcode the `/build` directory?  Is this consistently mapped in all build environments?

Reply:
/build is [our BUILD_BASE](https://github.com/ManageIQ/manageiq-appliance-build/blob/master/scripts/vmbuild.rb#L33), so I think that's the safest place.